_studio/mfx_lib/CMakeLists.txt

```diff
@@ -164,7 +164,6 @@ endif()
 
```

Comment:
That's the first patch of this PR since all prev. 5 patches actually belong to another PR: https://github.com/Intel-Media-SDK/MediaSDK/pull/502. I based current PR on that one since they significantly overlap in top level cmake and readme.